uaa/src/test/java/org/cloudfoundry/identity/uaa/login/AccountsControllerMockMvcTests.java

```diff
@@ -206,7 +206,7 @@ void testDisableSelfServiceCreateAccountPost() throws Exception {
                 .param("email", userEmail)
                 .param("password", "secr3T")
                 .param("password_confirmation", "secr3T"))
-                .andExpect(model().attribute("error_message_code", "self_service_disabled"))
+                .andExpect(model().attribute("error_message_code", "self_service_create_account_disabled"))
```

Comment:
Add similar tests to [ResetPasswordControllerMockMvcTests](https://github.com/GESoftware-CF/uaa/blob/predix_extensions_75.18.1/uaa/src/test/java/org/cloudfoundry/identity/uaa/login/ResetPasswordControllerMockMvcTests.java):
- testResetPasswordWithDisableSelfService()
- testDisableSelfServiceResetPasswordPost()